pkg/rpc/context.go

```diff
@@ -2322,6 +2327,51 @@ func (rpcCtx *Context) runHeartbeat(
 	}
 }
 
+func (rpcCtx *Context) VerifyDialback(ctx context.Context, nodeID roachpb.NodeID) error {
+	log.Errorf(ctx, "Verifying health of connection to n%d", nodeID)
+	if nodeID == 0 {
+		//FIXME: at startup, the nodeID might not be set. Unfortunately
```

Comment:
This only happens when bootstrapping a new node, right? Once the node has joined a cluster, this will always be set even on the initial connection attempts? If so, this seems totally fine, since the node can't participate in consensus or acquire any leases before it's allocated a node ID.